gateway/src/apicast/policy/rewrite_url_captures/named_args_matcher.lua

```diff
@@ -99,16 +99,35 @@ end
 --   with "{}". For example: "/{var_1}/something/{var_2}".
 -- @tparam string template Template in which the named args matched will be
 --   replaced. For example: "/v2/something/{var_1}?my_arg={var_2}".
-function _M.new(match_rule, template)
+function _M.new(match_rule, template, methods)
   local self = setmetatable({}, mt)
 
   self.named_args = extract_named_args(match_rule)
   self.regex_rule = transform_rule_to_regex(match_rule)
   self.template = template
+  self.methods = methods
 
   return self
 end
 
+-- Returns true if the Method of the request is in the methods of the command meaning the rewrite rule should be applied
+-- Returns true if no Method is provided in the config for backwardscompatibility
+local function is_match_methods(methods)
+
+  local request_method = ngx.req.get_method()
```

Comment:
I would move this line after the condition, but it's ok if you want to keep it.

Reply:
ah yes makes sense, not using it in the conditional statement anyway, thanks